src/Surfnet/StepupRa/RaBundle/Resources/views/SecondFactor/auditLog.html.twig

```diff
@@ -32,7 +32,7 @@
                     <td>{{ logEntry.secondFactorIdentifier }}</td>
                     <td>{{ logEntry.secondFactorType }}</td>
                     <td>{{ ('ra.auditlog.action.' ~ logEntry.action)|trans }}</td>
-                    <td>{{ logEntry.recordedOn|date("Y-m-d H:i") }}</td>
+                    <td><time datetime="{{ logEntry.recordedOn.format('c') }}">{{ logEntry.recordedOn.format("Y-m-d H:i") }}</time></td>
```

Comment:
As we're now properly handling these dates, what about adding something like "UTC" to the default `<time>` text?

:+1:, though!